services/migration_service/api/__init__.py

```diff
@@ -9,19 +10,22 @@
 
 latest = "latest"
 
-goose_template = "goose postgres " \
-                 "\"dbname={} " \
-                 "user={} " \
-                 "password={} " \
-                 "host={} " \
-                 "port={} " \
-                 "sslmode=disable\" {}"
+def make_goose_template(conn_str,command):
+    return ' '.join(shlex.quote(arg) for arg in [
+        "goose",\
+        "postgres",\
+        f"\"{conn_str}\"",\
+        f"{command}"
+    ])
 
 path = os.path.dirname(__file__) + "/../migration_files"
-goose_migration_template = "goose -dir " + path + " postgres " \
-    "\"dbname={} " \
-    "user={} " \
-    "password={} " \
-    "host={} " \
-    "port={} " \
-    "sslmode=disable\" {}"
+
+def make_goose_migration_template(conn_str,command):
+    return ' '.join(shlex.quote(arg) for arg in [
+        "goose",\
+        "-dir",\
+        path,\
+        "postgres",\
+        f"\"{conn_str}\"",\
```

Comment:
i don't think you need the quotes here, shlex will add them if necessary
```suggestion
        conn_str,
```

Reply:
Lemme test that

Reply:
Done.